e2e/appActions.js

```diff
@@ -29,19 +29,46 @@
  * timer.e2e.spec.js and notebook.e2e.spec.js, that function should be generalized and moved into this file.
  */
 
+/**
+ * Uniquely identifies a domain object.
+ *
+ * @typedef Identifier
```

Comment:
Can you document this documentation? This team is relatively unfamiliar with what this accomplishes

Reply:
Sure, I can elaborate here. This snippet was actually lifted directly out of our `ObjectAPI`. I figured we should use the type if it has already been defined somewhere.

Reply:
@ozyx can we reference the types from that object API?

Reply:
We could! The only issue is that we would lose them when we pull the opensource tests down to VIPER side of things. But now as I think about it, this should be fine since we aren't developing there, we are just going to be running the tests. I'll update this.